### PR TITLE
feat: add Base and Base Sepolia

### PR DESCRIPTION
**Pull request by gleiser-oliveira**
## Changes

- Update Venus packages and allow Base/Base Sepolia to be selected

### CODE REVIEW

File: packages/chains/src/chainMetadata.ts [hunk 1]
```diff
@@ -144,4 +145,21 @@ export const chainMetadata: {
...
+    explorerUrl: 'https://base.blockscout.com',
+    layerZeroScanUrl: 'https://testnet.layerzeroscan.com',
+    // add BASE_MAINNET core comptroller address
+    corePoolComptrollerContractAddress: '',
+    nativeToken: ethToken,
+  },
+  [ChainId.BASE_SEPOLIA]: {
+    name: 'Base Sepolia',
+    logoSrc: baseLogo,
+    explorerUrl: 'https://base-sepolia.blockscout.com',
```
Comment (chechu) [inline packages/chains/src/chainMetadata.ts]:
```suggestion
    explorerUrl: 'https://sepolia.basescan.org',
```

File: packages/chains/src/chainMetadata.ts [hunk 2]
```diff
@@ -144,4 +145,21 @@ export const chainMetadata: {
     corePoolComptrollerContractAddress: '0x59d10988974223B042767aaBFb6D926863069535',
     nativeToken: ethToken,
   },
+  [ChainId.BASE_MAINNET]: {
+    name: 'Base',
+    logoSrc: baseLogo,
+    explorerUrl: 'https://base.blockscout.com',
```
Comment (chechu) [inline packages/chains/src/chainMetadata.ts]:
```suggestion
    explorerUrl: 'https://basescan.org',
```

File: apps/evm/src/libs/contracts/config/index.ts [hunk 3]
```diff
@@ -418,6 +452,8 @@ export const contracts: ContractConfig[] = [
       [ChainId.ZKSYNC_MAINNET]: tokenBridgeZkSyncMainnetDeployments.addresses.XVSProxyOFTDest,
       [ChainId.OPTIMISM_SEPOLIA]: tokenBridgeOptimismSepoliaDeployments.addresses.XVSProxyOFTDest,
       [ChainId.OPTIMISM_MAINNET]: tokenBridgeOptimismMainnetDeployments.addresses.XVSProxyOFTDest,
+      [ChainId.BASE_MAINNET]: tokenBridgeBaseSepoliaDeployments.addresses.XVSProxyOFTDest,
```
Comment (therealemjy) [inline apps/evm/src/libs/contracts/config/index.ts]:
```suggestion
      [ChainId.BASE_MAINNET]: tokenBridgeBaseMainnetDeployments.addresses.XVSProxyOFTDest,
```

**Review by therealemjy — COMMENTED** [review]
We'll just need to add the subgraph URL for Base mainnet.

File: apps/evm/src/libs/contracts/config/index.ts (hunk 3 above)
Comment (gleiser-oliveira) [reply apps/evm/src/libs/contracts/config/index.ts]: Thanks!!

**Review by chechu — APPROVED** [review]

**Pull request merged**